fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityTypeBuilder.java

```diff
@@ -63,7 +100,7 @@ protected FabricEntityTypeBuilder(SpawnGroup spawnGroup, EntityType.EntityFactor
 	 * @return a new entity type builder
 	 */
 	public static <T extends Entity> FabricEntityTypeBuilder<T> create(SpawnGroup spawnGroup) {
-		return new FabricEntityTypeBuilder<>(spawnGroup, (t, w) -> null);
+		return create(spawnGroup, FabricEntityTypeBuilder::empty);
```

Comment:
Is this a reasonable default ? It would probably lead to a lot of modded entities being unable to be summoned without authors realizing it.

Reply:
That's also the current behaviour of the method, so changing it could break existing mods.

Reply:
I realize that, but maybe this PR is a good opportunity to deprecate this default.